src/freenet/node/PeerNode.java

```diff
@@ -3258,7 +3258,7 @@ public void addToLocalNodeSentMessagesToStatistic(Message m) {
 			if(count == null)
```

Comment:
`Map.merge()` calls again, for the one below as well.

Reply:
I will look into it another time. Please don't hold up the PR waiting for me.